modules/antlr4-cpp/4.12.0/MODULE.bazel

```diff
@@ -0,0 +1,12 @@
+module(
+    name = "antlr4-cpp",
```

Comment:
Does it make sense to include the version number in the name? (Not saying it doesn't, just want to start a discussion on the potential duplication).

Reply:
Sounds good, but shouldn't we also give the target the same name then? Or at least add an alias.